.github/workflows/c3-e2e-dependabot.yml

```diff
@@ -66,13 +66,19 @@ jobs:
   # For dependabot versioning PRs we only want to run the e2es for the specifically bumped
   # framework (this is both for optimization and in order to reduce unnecessary flakiness)
   e2e-only-dependabot-bumped-framework:
+    # Note: please keep this job in sync with the e2e one
```

Comment:
maybe we could find a way to reuse this code (like a shared parametrized local action)

I can look into that but it I am not sure if it's overkill here, please let me know what you think

Reply:
I think its OK for now.